lib/shopify-cli/helpers/partners_api.rb

```diff
@@ -0,0 +1,66 @@
+require 'shopify_cli'
+
+module ShopifyCli
+  module Helpers
+    class PartnersAPI < API
+      ENV_VAR = 'SHOPIFY_APP_CLI_LOCAL_PARTNERS'
+      AUTH_PROD_URI = 'https://accounts.shopify.com'
+      AUTH_DEV_URI = 'https://identity.myshopify.io'
+      PROD_URI = 'https://partners.shopify.com'
+      DEV_URI = 'https://partners.myshopify.io/'
+      PROD_ID = '28e63026-d1ad-4526-9640-bc85b5ae88e2'
+      DEV_ID = 'df89d73339ac3c6c5f0a98d9ca93260763e384d51d6038da129889c308973978'
+      PROD_CLI_ID = 'fbdb2649-e327-4907-8f67-908d24cfd7e3'
+      DEV_CLI_ID = 'e5380e02-312a-7408-5718-e07017e9cf52'
+
+      class << self
+        def id
+          ENV[ENV_VAR].nil? ? PROD_ID : DEV_ID
```

Comment:
I wonder if using `ShopfiyCli::Config` would be better for this, which reads from a `config.ini` in the home directory. This is a cli-kit thing so far only being used to enable autoupdates: https://github.com/Shopify/shopify-app-cli/blob/dd275f0472a72e946c36feec76a2e7af48f934f4/lib/shopify-cli/update.rb#L11

What do you think?

Reply:
It seems like that persists, but I like the fact that with the environment variable, you can export it in the current shell and work locally but if you open a new window/tab it will default back to the production configs